src/Sql/Sql/DistributedAvailabilityGroup/Cmdlet/GetAzureSqlManagedInstanceLink.cs

```diff
@@ -0,0 +1,147 @@
+﻿using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.Sql.ManagedInstance.Model;
+using Microsoft.Azure.Commands.Sql.ManagedInstanceHybridLink.Model;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Management.Automation;
+using System.Text;
+
+namespace Microsoft.Azure.Commands.Sql.ManagedInstanceHybridLink.Cmdlet
+{
+    /// <summary>
+    /// Cmdlet to get a Manged Instance Link
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "SqlInstanceLink",
+        DefaultParameterSetName = GetByNameParameterSet
+        ),
+        OutputType(typeof(AzureSqlManagedInstanceLinkModel))]
+    public class GetAzureSqlManagedInstanceLink : AzureSqlManagedInstanceLinkCmdletBase
+    {
+        private const string GetByNameParameterSet = "GetByNameParameterSet";
+        private const string GetByParentObjectParameterSet = "GetByParentObjectParameterSet";
+        private const string GetByResourceIdParameterSet = "GetByResourceIdParameterSet";
+        private const string GetByInstanceResourceIdParameterSet = "GetByInstanceResourceIdParameterSet";
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(Mandatory = true, ParameterSetName = GetByNameParameterSet, Position = 0, HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of target managed instance
+        /// </summary>
+        [Parameter(Mandatory = true, ParameterSetName = GetByNameParameterSet, Position = 1, HelpMessage = "The name of the Azure SQL Managed Instance.")]
+        [ResourceNameCompleter("Microsoft.Sql/managedInstances", nameof(ResourceGroupName))]
+        [ValidateNotNullOrEmpty]
+        public string InstanceName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the link name
+        /// </summary>
+        [Parameter(Mandatory = false, ParameterSetName = GetByNameParameterSet, Position = 2, HelpMessage = "The name of the Managed Instance link.")]
+        [Parameter(Mandatory = false, ParameterSetName = GetByParentObjectParameterSet, Position = 1, HelpMessage = "The name of the Managed Instance link.")]
+        [Parameter(Mandatory = false, ParameterSetName = GetByInstanceResourceIdParameterSet, Position = 1, HelpMessage = "The name of the Managed Instance link.")]
+        [ResourceNameCompleter("Microsoft.Sql/managedInstances/distributedAvailabilityGroups", nameof(ResourceGroupName), nameof(InstanceName))]
+        [ValidateNotNullOrEmpty]
+        public string LinkName { get; set; }
```

Comment:
Please use Name as the official name of LinkName and keep LinkName as alias to follow [parameter standard name convention](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/parameter-best-practices.md#standard-parameter-name)

Reply:
Sure, thanks